Add conceptual docs for react-aria (#471)
adobe-private/react-spectrum-v3: packages/@react-aria/interactions/docs/interactions.mdx

key continues to navigate to the next/previous tab stop. React Aria implements Hooks for many of these composite components and handles 
all of the keyboard navigation behavior for elements inside them.

Overlay elements like dialogs, popovers, and menus have additional focus behavior to ensure that focus stays within them while they
are open, and focus is restored back to the element that opened them when they are closed. In React Aria this is implemented by
the [FocusScope](FocusScope.html) component.

Another important feature for keyboard users is a **focus ring**. This is a visual affordance for the currently focused element, which
allows a keyboard user to know which element they are currently on. It should only be visible when navigating with a keyboard, so as
not to distract mouse and touchscreen users. This can be implemented using the [useFocusRing](useFocusRing.html) hook or the 
[FocusRing](FocusRing.html) component. The [useFocusVisible](useFocusVisible.html) hook can also be used to determine whether
the user is currently navigating with a keyboard or not.

## Assistive technology

> Assistive technology like screen readers rely on semantic information exposed to the browser through ARIA or native HTML element

Reviewer: ```suggestion
An Assistive technology such as a screen reader relies on semantic information exposed to the browser through ARIA or native HTML element
```
